tox.ini

```diff
@@ -0,0 +1,50 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = True
+envlist =
+    py{35,36,37,py3}
```

Comment:
38?

Reply:
> Can you expand on why you switched out unites for Pytest?

We switched out unittest for pytest since aiohttp’s testing modules support pytest directly. Additionally, pytest brings advantages such as parallel testing and parameterizing of tests. Parameterizing of tests means that we can run a test with multiple inputs to determine how they might react given a predefined set of values instead of duplicating the test itself. Pytest also comes with fixtures which allow us to build up and teardown environments to ensure state between dynamic test(such as those that would require webserver interaction)

Reply:
aiohttp does support unittest too https://docs.aiohttp.org/en/stable/testing.html#unittest 

That said I do personally like pytest more.

Reply:
@khyberspache Python 3.8 added